protected/humhub/modules/content/widgets/richtext/converter/BaseRichTextConverter.php

```diff
@@ -284,7 +287,7 @@ protected function renderAbsy($blocks)
             });
         }
 
-        return parent::renderAbsy($blocks);
+        return '';
```

Comment:
Why this change this will should break the whole parser process or am I missing something?

Reply:
I changed this because the `parent::renderAbsy($blocks)` is abstract method and IDE displays error, please review my second screenshot from this comment https://github.com/humhub/humhub/issues/4857#issuecomment-779682809.

Reply:
See https://github.com/humhub/humhub/issues/4857#issuecomment-779821434 this seems to be a problem with phpstorm. The method is overwritten in a parent class already.

Reply:
ok, I reverted that here https://github.com/humhub/humhub/pull/4882/commits/27e2022aebcf6c84b8098b715a9566dc47360075

Reply:
Just an idea and could be ignored here if a better method could be implemented

```php
return parent::renderAbsy($blocks['content']);
```
or
```php
return rtrim(parent::renderAbsy($blocks['content']));
```

Reply:
@buddh4 Do you agree to apply `rtrim()` there?

Reply:
No, please leave as is.